routes/api.php

```diff
@@ -32,10 +32,10 @@
     Route::post('/update','UserController@update');
     Route::post('/resetpassword', 'UserController@resetPassword');
     Route::post('/logout', 'UserController@logout');
-    Route::post('/getUserPost', 'PostController@getUserPost');
-    Route::post('/getAllPost', 'PostController@getAllPost');
-    Route::post('/setPost','PostController@setPost');
-    Route::post('/getLocationByDistance','PostController@getLocationByDistance');
+    Route::post('/getuserpost', 'PostController@getUserPost');
+    Route::post('/getallpost', 'PostController@getAllPost');
+    Route::post('/setpost','PostController@setPost');
+    Route::post('/getpostbylocation','PostController@getPostByLocation');
```

Comment:
remove those get and set from url. method will automatically define those things